src/driver/mysql/MysqlDriver.ts

```diff
@@ -37,6 +45,11 @@ export class MysqlDriver implements Driver {
      */
     mysql: any;
 
+    /**
+     * Mysql version.
+     */
+    version: MySQLVersion;
```

Comment:
I'm sorry to annoy you but we don't really need enum here. Regular string literal type is enough. (e.g. replace `version: MySQLVersion` to `version: "mysql"|"mysql2")